web_domain_field/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Web Domain Field",
+    "summary": """
+        Use computed field as domain""",
+    "version": "16.0.1.0.1",
```

Comment:
As per guideline (https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#tasks-to-do-in-the-migration)
```suggestion
    "version": "16.0.1.0.0",
```